src/sentry/web/frontend/auth_organization_id_login.py

```diff
@@ -0,0 +1,27 @@
+from django.db import transaction
+from django.http import HttpResponse
+from django.urls import reverse
+from django.views.decorators.cache import never_cache
+from rest_framework.request import Request
+
+from sentry.services.hybrid_cloud.organization import organization_service
+from sentry.web.frontend.auth_organization_login import AuthOrganizationLoginView
+
+
+class AuthOrganizationIdentifierLoginView(AuthOrganizationLoginView):
+    @never_cache
+    @transaction.atomic
```

Comment:
Does this need to be atomic? we're not writing anything right?

Reply:
The parent handle method is which we're calling here but if I redirect as the below comment suggests then no, this is not needed either.